lua/bible-verse/core/init.lua

```diff
@@ -110,4 +111,32 @@ function M.query_and_insert()
 	Ui:input(input_conf, on_submit)
 end
 
+--- Returns a random verse
+function M.random_query()
+	return process_query(random_verse(), Config.options.query_format, 0)
```

Comment:
I'm not sure whether it would be easy to use `M.random_query()` if the format is using `Config.options.query_format` 🤔 

A user who has set `Config.options.query_format=bibleverse` to get highlighting on the pop up will get difficult to read/use output from `M.random_query()`. 

Do you think it would be better to create a separate `Config.options`?

Reply:
It's true, I haven't played enough with the config to know about this. I'll let you be the judge, it's probably a good idea :)